plaaces-to-go/src/styles/Menu.module.css

```diff
@@ -0,0 +1,100 @@
+.navbar{
+    display: flex;
+    align-items: center;
+    background: #f7e9da;
+}
+
+.navLogo{
+    display: flex;
+    flex: 1;
+}
+
+.txtItem{
+    color: #3f6b7c;
+    font-size: 20px;
+    padding: 20px;
+    font-family: 'Lato', 'Arial Narrow', Arial, sans-serif;
+    font-weight: 800;
+}
+
+.navItems{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    flex: 2;
+    color: #71899b;
+    text-decoration: none;
+    list-style-type: none;
+    padding: 0 20px;
+    font-weight: 700;
+    margin: 0;
+}
+
+.navItems li:hover{
+    color: #b97479;;
+}
+.navItems li{
+    padding: 0 15px;
+}
+
+.navToggle {
+    display: none;
+}
+
+@media (max-width: 768px) {
+    .navItems{
+        position: absolute;
```

Comment:
Are you doing this to have the items below the header? You could try to make it happen with relative positioned elements, generally absolute positioned elements should be the last resource, you should prefer to depend on the implicit styles elements have, menu is already the top-most element, no need to position it absolutely on the top, but I could be wrong and you may need it to make the transforms work, let me know